### PR TITLE
Create SignedBinaryProxyExecutionRundll32.yaml

### PR DESCRIPTION
**Pull request by chihebchebbi**
A new  rule to hunt Rundll32 activities

### CODE REVIEW

File: Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  - connectorId: SecurityEvents
+    dataTypes:
+      - SecurityEvent
+tactics:
+  - Defense Evasion
+relevantTechniques:
+  - T1218.011
+
+query: |
+  let timeframe = 1d;
```
Comment (shainw) [inline Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml:16]: Remove, see explanation in overall comments.

File: Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  Event
+  //This query uses sysmon data depending on table name used this may need updataing
+  | where TimeGenerated >= ago(timeframe)
```
Comment (shainw) [inline Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml:19]: Remove, see explanation in overall comments.

**Review by shainw — CHANGES_REQUESTED** [review]
We are not including timeframes any longer when there is no comparison lookback.  This is to support the Hunting UX and how hard coded timeframes can cause issues with running queries automatically.  Please remove ```let timeframe = 1d;``` and ```| where TimeGenerated >= ago(timeframe)```

File: Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+query: |
+  Event
+  //This query uses sysmon data depending on table name used this may need updataing
+  | where Source == "Microsoft-Windows-Sysmon"
+  | where EventID == 1
+  | extend RenderedDescription = tostring(split(RenderedDescription, ":")[0])
+  | extend EventData = parse_xml(EventData).DataItem.EventData.Data
+  | mv-expand bagexpansion=array EventData
+  | evaluate bag_unpack(EventData)
+  | extend Key=tostring(['@Name']), Value=['#text']
```
Comment (shainw) [inline Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml:24]: This query will fail execution and rule creation if we don't have a recent event or an event within the timeframe.  Changing this line to - `| extend Key = tostring(column_ifexists('@Name', "")), Value = column_ifexists('#text', "")` - will allow it to not fail.

**Review by shainw — CHANGES_REQUESTED** [review]
One more item I missed previously.  Quick fix and then good to go.

File: Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml (hunk 3 above)
Comment (chihebchebbi) [reply Hunting Queries/SecurityEvent/SignedBinaryProxyExecutionRundll32.yaml]: Done. Thank you @shainw

**Review by shainw — APPROVED** [review]

**Pull request merged**